.github/workflows/test.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
   test:
     name: ${{ matrix.os }} ${{ matrix.python-version }}
     runs-on: ${{ matrix.os }}
-
+    timeout-minutes: 60
```

Comment:
Right now, the CI job finishes in around six-ish minutes. On the master branch, including the `test_training.py` script, it takes over half an hour. Google says the default github actions timeout is 360 minutes (6 hours)... but we were seeing CI jobs killed with an error much sooner than that on the master branch (maybe they ran out of memory?)

I'm not sure whether we should keep this line, remove it, or change it to a different number. I'm leaning towards keeping it in here, and maybe changing the timeout number if that makes sense.

Reply:
Let's keep it for now. I will look into the test.